package.json

```diff
@@ -9,7 +9,7 @@
     "url": "https://github.com/magento-research/pwa-studio/issues"
   },
   "engines": {
-    "node": ">=8.0.0"
+    "node": ">=10.3.0"
```

Comment:
So we may want to actually bump this to `10.14.1` since that is the latest LTS version of node
https://nodejs.org/en/download/releases/

Reply:
This is saying we "work on `10.3.0` and above" so I think this is good 👍

Reply:
Outdated already! 😃